pkg/query/endpointset_test.go

```diff
@@ -269,13 +271,54 @@ func (e *testEndpoints) CloseOne(addr string) {
 	delete(e.srvs, addr)
 }
 
+func TestTruncateExtLabels(t *testing.T) {
+	testCases := []struct {
+		name            string
+		labelToTruncate string
+		lengthLimit     int
+		expectedOutput  string
+	}{
+		{
+			name:            "shorter label length",
+			labelToTruncate: "{xxx}",
+			lengthLimit:     5,
+			expectedOutput:  "{xxx}",
```

Comment:
having `xxx` is not great as it's repetitive so we don't know if the function will not cut in wrong moment.

Reply:
fixed